### ISSUE TITLE
lantern test fails (IndexError: list index out of range)

### ISSUE THREAD

**Issue by anadahz** [TheTorProject/ooni-probe#446]
```
ooniprobe -ng /usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/nettests/third_party/lantern.py -u http://www.google.com
```
```
[-] Not reporting using a collector
[-] [D] No test deck detected
[-] [D] Checking if url is present
[-] You should add annotations for the country, city and ASN
[-] Fetching required net test inputs...
[-] [D] Fetching and verifying inputs
[-] Looking up collector and test helpers
[-] [D] Creating /root/report-lantern_circumvention_tool_test-2015-12-10T172439Z.yamloo
[-] [D] Writing report with YAML reporter
[-] [D] Starting this task <generator object generateMeasurements at 0x7f334354c870>
[-] [D] Running <class 'ooni.nettests.third_party.lantern.LanternTest'> test_lantern_circumvent
[-] [D] Spawning Lantern
[-] [D] Task <ooni.tasks.Measurement object at 0x7f3343550b10> has failed 1 times
[-] Traceback (most recent call last):
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/managers.py", line 61, in _fillSlots
[-]     self._run(task)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/managers.py", line 77, in _run
[-]     d = task.start()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 92, in start
[-]     return BaseTask.start(self)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 42, in start
[-]     self._running = defer.maybeDeferred(self.run)
[-] --- <exception caught here> ---
[-]   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 150, in maybeDeferred
[-]     result = f(*args, **kw)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 145, in run
[-]     return self.netTestMethod()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/nettests/third_party/lantern.py", line 99, in test_lantern_circumvent
[-]     self.runLantern()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/nettests/third_party/lantern.py", line 75, in runLantern
[-]     reactor.spawnProcess(self.processDirector, paths[0], self.command)
[-] exceptions.IndexError: list index out of range
[-] [D] Rescheduling...
[-] [D] Spawning Lantern
[-] [D] Task <ooni.tasks.Measurement object at 0x7f3343550b10> has failed 2 times
[-] Traceback (most recent call last):
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/managers.py", line 61, in _fillSlots
[-]     self._run(task)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/managers.py", line 77, in _run
[-]     d = task.start()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 92, in start
[-]     return BaseTask.start(self)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 42, in start
[-]     self._running = defer.maybeDeferred(self.run)
[-] --- <exception caught here> ---
[-]   File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 150, in maybeDeferred
[-]     result = f(*args, **kw)
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/tasks.py", line 145, in run
[-]     return self.netTestMethod()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/nettests/third_party/lantern.py", line 99, in test_lantern_circumvent
[-]     self.runLantern()
[-]   File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/ooni/nettests/third_party/lantern.py", line 75, in runLantern
[-]     reactor.spawnProcess(self.processDirector, paths[0], self.command)
[-] exceptions.IndexError: list index out of range
[-] [D] Rescheduling...
[-] [D] Spawning Lantern
[-] [D] Task <ooni.tasks.Measurement object at 0x7f3343550b10> has failed 3 times
[-] Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1438, in run_script
    execfile(script_filename, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/EGG-INFO/scripts/ooniprobe", line 13, in <module>
    @d.addBoth
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 340, in addBoth
    callbackKeywords=kw, errbackKeywords=kw)
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 306, in addCallbacks
    self._runCallbacks()
--- <exception caught here> ---
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/defer.py", line 588, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/local/lib/python2.7/dist-packages/ooniprobe-1.3.2-py2.7.egg/EGG-INFO/scripts/ooniprobe", line 15, in cb
    reactor.stop()
  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/base.py", line 581, in stop
    "Can't stop reactor that isn't running.")
twisted.internet.error.ReactorNotRunning: Can't stop reactor that isn't running.
```

**Comment by hellais** [comment]
@aagbsn can you look at this.

**Comment by anadahz** [comment]
This issue has been fixed in #447
